tests/test_save_identity.py

```diff
@@ -14,6 +12,7 @@ def test_save_identity(self):
         re = self.client.post('/v1/identities', data=json.dumps(payload))
         self.assertEqual(200, re.status_code)
 
+    # TODO: test failure scenarios
 
 if __name__ == '__main__':
-    unittest.main()
+    unittest.main()
```

Comment:
`pycodestyle` is complaining about this change:

```
./tests/test_save_identity.py:18:20: W292 no newline at end of file
```

It's good practice to finish all lines with newline. That way, adding a new line after this line won't trigger git.